site/data/sidebar.yml

```diff
@@ -104,6 +104,7 @@
     - title: Clearfix
     - title: Color & background
     - title: Colored links
+    - title: Focus ring
```

Comment:
FYI moved the page here from 'Utilities' to 'Helpers' so that we can see it in the Netlify deployment pages and to avoid an error for the Docs GitHub action.